webvtt.html

```diff
@@ -542,6 +542,17 @@
    </section>
 
    <section>
+    <h3>Unicode normalization</h3>
+
+    <p>This specification does not normalize Unicode text as part of its processing
+    requirements.</p>
+
+    <p>For example, a cue with the identifier consisting of the LATIN CAPITAL LETTER A followed by
```

Comment:
Maybe "consisting of the characters U+0041 LATIN CAPITAL LETTER A and U+212B ANGSTROM SIGN"?